source/CF/Constants.pm

```diff
@@ -267,7 +267,7 @@ sub list_clusterflow_genomes {
 
     my $returnstring = "";
 
-    my @config_files = ("$FindBin::Bin/genomes.config", "$homedir/.clusterflow/genomes.config", './genomes.config');
+    my @config_files = ("$FindBin::Bin/genomes.config","$FindBin::Bin/genomes.d/","$homedir/.clusterflow/genomes.config","$homedir/.clusterflow/genomes.d/",'./genomes.config');
```

Comment:
No. I did it that way initially but the output is a mess as it lists every file separately. Doing it this way reports at the level of the directory. 

On 17 Jun 2016 5:10 p.m., Phil Ewels <notifications@github.com> wrote:

In source/CF/Constants.pm:

> @@ -267,7 +267,7 @@ sub list_clusterflow_genomes { > > my $returnstring = ""; > > - my @config_files = ("$FindBin::Bin/genomes.config", "$homedir/.clusterflow/genomes.config", './genomes.config'); > + my @config_files = ("$FindBin::Bin/genomes.config","$FindBin::Bin/genomes.d/","$homedir/.clusterflow/genomes.config","$homedir/.clusterflow/genomes.d/",'./genomes.config'); 

Does this need to be glob("$FindBin::Bin/genomes.d/*config") as above instead of just "$FindBin::Bin/genomes.d/"?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.